packages/babel-plugin-transform-es2015-block-scoping/src/index.js

```diff
@@ -100,6 +100,18 @@ function isBlockScoped(node) {
   return true;
 }
 
+/**
+ * If there is a loop ancestor closer than the closest function, we
+ * consider ourselves to be in a loop.
+ */
+function isInLoop(path) {
+  const loopOrFunctionParent = path.find(
+    path => path.isLoop() || path.isFunction(),
```

Comment:
We probably need to check for `#isProgram`, too.

Reply:
Looks like it'll just be falsy if it gets to the top so that is addressed?